### PR TITLE
XIVY-14003 new icons

### PR DESCRIPTION
**Pull request by ivy-lgi**
icon for drag and drop reorder in table
icon for empty details view of variables editor

Old:
image

New:
image

### CODE REVIEW

File: packages/icons/src/icons/edit-dots copy.svg
Comment (weissreto) [inline packages/icons/src/icons/edit-dots copy.svg:1]: Does this file need to be named like this (copy)? 
Shouldn't it be edit-dots.svg

Comment (ivy-lgi) [reply packages/icons/src/icons/edit-dots copy.svg:1]: my bad, the icon was already present :sweat_smile:

File: packages/icons/src/icons/drag-drop.svg
Comment (weissreto) [inline packages/icons/src/icons/drag-drop.svg:1]: Why add this icon if we don't use it anywhere?

**Review by weissreto — APPROVED** [review]
Another small question. Otherwise good.

File: packages/icons/src/icons/drag-drop.svg
Comment (ivy-lgi) [reply packages/icons/src/icons/drag-drop.svg:1]: the variables editor gets its icons from here, it will be used there.

Comment (ivy-lgi) [reply packages/icons/src/icons/drag-drop.svg:1]: See https://github.com/axonivy/config-editor-client/pull/109

**Pull request merged**